sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -81,6 +81,22 @@ extern "C" {
 #define GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US (3U * US_PER_SEC)
 #endif
 
+/**
+ * @brief   Keep all but oldest fragment when reassembly buffer is full
+ *
+ * @note    Only applicable with
+ *          [gnrc_sixlowpan_frag_rb](@ref net_gnrc_sixlowpan_frag_rb) module
+ *
+ * When not set, it will cause the reassembly buffer to
+ * override the oldest entry no matter what. When set, only the oldest
```

Comment:
```suggestion
 * override the oldest entry if new a entry has to be created and the reassembly buffer is full, no matter what. When set, only the oldest
```

Also please fix (your and my) line breaks.